tests/untl_structure_test.py

```diff
@@ -409,12 +401,12 @@ def test_Metadata_create_xml_string():
     description = us.Description(content=u'Adaption of "Fortuna te dé Dios, hijo"',
                                  qualifier=u'content')
     metadata.children = [title, description]
-    expected_text = """<?xml version="1.0" encoding="UTF-8"?>
+    exp_text = """<?xml version=\'1.0\' encoding=\'utf-8\'?>
```

Comment:
Why was this change made? Double-quotes aren't a problem within triple-quoted string literals. Also, for those times when they _are_ a problem, the solution usually is to _either_ escape the quotes with a backslash _or_ to use the other type of quotes. It's superfluous to do _both_.

Reply:
I wanted to use single quotes and was trying to escape them as well while making modifications, but forgot to change it back and it didn't raise any error in the test so, I didn't realize it was there. Got rid of the escape sequences. Thanks:)

Reply:
Ok, so to continue the conversation from the other thread, I hadn't realized that our XML encoding declaration had _changed_ because we're now using the `xml_declaration=True` argument to `tostring()`. Now what comes out uses single quotes and lower-case "utf-8". So yeah, just get rid of the escaping backslashes on all the instances of this declaration and they should be fine.